packages/application-shell-connectors/src/components/application-context/application-context.spec.tsx

```diff
@@ -41,12 +45,21 @@ describe('rendering', () => {
             countries: ['us'],
             currencies: ['USD'],
             languages: ['en'],
-            permissions: { canManageProjectSettings: true },
-            actionRights: null,
-            dataFences: null,
+            allAppliedPermissions: [
```

Comment:
The shape of the user/project that we pass to the context provider now maps 1:1 to the query response